text/0000-crates.io-default-ranking.md

```diff
@@ -0,0 +1,1440 @@
+- Feature Name: crates_io_default_ranking
+- Start Date: 2016-12-19
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Crates.io has many useful libraries for a variety of purposes, but it's
+difficult to find which crates are meant for a particular purpose and then to
+decide among the available crates which one is most suitable in a particular
+context. [Categorization][cat-pr] and [badges][badge-pr] are coming to
+crates.io; categories help with finding a set of crates to consider and badges
+help communicate attributes of crates. The question of how to order crates
+within a category, or within the list of crates that have a particular keyword,
+is still open. This RFC proposes a method of ranking crates combining number of
+downloads, version, and other attributes in order to help people decide what
+crate to use.
+
+[cat-pr]: https://github.com/rust-lang/crates.io/pull/473
+[badge-pr]: https://github.com/rust-lang/crates.io/pull/481
+
+# Motivation
+[motivation]: #motivation
+
+Finding and evaluating crates can be time consuming. People already familiar
+with the Rust ecosystem often know which crates are best for which puproses, but
+we want to share that knowledge with everyone. For example, someone looking for
+a crate to help create a parser should be able to navigate to a category
+for that purpose and get a list of crates to consider. This list would include
+crates such as [nom][] and [peresil][], and the order in which they appear
+should be significant and should help make the decision between the crates in
+this category easier.
+
+[nom]: https://crates.io/crates/nom
+[peresil]: https://github.com/docopt/docopt.rs
```

Comment:
This links the wrong library and can be confusing. I looked up the library mentioned and it makes more sense since this is a parsing library.
https://github.com/shepmaster/peresil